main.go

```diff
@@ -105,11 +119,13 @@ func addCommentToIssue(ctx context.Context, post sfapi.DiscussionPost, ticket *s
 func addCommentsToIssue(ctx context.Context, progressDB ProgressState, ticket *sfapi.Ticket, issue *github.Issue) error {
 	if len(ticket.DiscussionThread.Posts) > 0 {
 		for _, post := range ticket.DiscussionThread.Posts {
-			if stopped {
+			if stopped && ! cliConfig.skipcomments {
+				// do not exit in the middle of comments if skipcomments is enabled
 				return nil
 			}
 
 			if _, found, _ := progressDB.Get("comment", post.Slug); found {
+		                verbose(fmt.Sprintf("  Comment already exists, skipping"))
```

Comment:
```suggestion
		                verbose("  Comment already exists, skipping")
```

Same, no need of Sprintf